packages/dashboard/src/components/RegisterFrame/ButtonGroup.tsx

```diff
@@ -0,0 +1,20 @@
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { buttonClasses, Stack } from '@material-ui/core'
+import type { PropsWithChildren } from 'react'
+
+const ButtonGroupUI = styled(Stack)(({ theme }) => ({
+    [`& > .${buttonClasses.root}`]: {
+        width: 225,
+        fontSize: 16,
+    },
+}))
+
+interface ButtonGroupProps extends PropsWithChildren<{}> {}
+
+export const ButtonGroup = ({ children }: ButtonGroupProps) => {
+    return (
+        <ButtonGroupUI my={7} direction="row" spacing={2} justifyContent="center">
```

Comment:
Why not use the real ButtonGroup? https://next.material-ui.com/components/button-group/

Reply:
Although their names are the same, but the UI style gap is large 😂 . ButtonGroup is in close proximity to each other in material-ui. But this component is not.